### PR TITLE
Add OS parameters for SLES

### PR DESCRIPTION
**Pull request by maximiliankolb**
This PR adds parameters for SLES operating systems.

Cherry-pick into:

* [x] Foreman 3.1

### CODE REVIEW

File: guides/common/modules/proc_creating-a-suse-operating-system.adoc [hunk 1]
```diff
@@ -15,6 +15,14 @@ The content of `/var/www/html/pub/` is publicly available on `\http://{foreman-e
 If you extract the `.iso` image to `/var/www/html/pub/installation_media/sles/15sp3/`, the path is `\http://{foreman-example-com}/pub/installation_media/sles/15sp3/`.
 . Set the *Operating System Family* to `SUSE` for all SLES systems.
 . Set a location and organization context for the installation media.
+. On the *Parameters* tab, add the `sle-module-basesystem-url` parameter as string with the value `\http://{foreman-example-com}/pub/installation_media/sles/15sp3/`.
+Note that the path depends on the extracted `.iso` image.
+ifdef::orcharhino[]
++
+Add the `or_client_repo_url` parameter as string with the value `\http://{foreman-example-com}/pulp/repos/_My_Organization_/Library/custom/SLES_Client/SLES_Client_15SP3/`.
```
Comment (melcorr) [inline guides/common/modules/proc_creating-a-suse-operating-system.adoc:22]:
```suggestion
Add the `or_client_repo_url` parameter, select the **string** type and enter the value `\http://{foreman-example-com}/pulp/repos/_My_Organization_/Library/custom/SLES_Client/SLES_Client_15SP3/`.
```
I think this is closer the the web UI if I am not mistaken?

File: guides/common/modules/proc_creating-a-suse-operating-system.adoc [hunk 2]
```diff
@@ -15,6 +15,14 @@ The content of `/var/www/html/pub/` is publicly available on `\http://{foreman-e
...
+endif::[]
++
+(SLES 12 only) Add the parameter `additional_media` as string with the value `""`.
```
Comment (melcorr) [inline guides/common/modules/proc_creating-a-suse-operating-system.adoc:25]: Same here as comment above.

File: guides/common/modules/proc_creating-a-suse-operating-system.adoc [hunk 3]
```diff
@@ -15,6 +15,14 @@ The content of `/var/www/html/pub/` is publicly available on `\http://{foreman-e
 If you extract the `.iso` image to `/var/www/html/pub/installation_media/sles/15sp3/`, the path is `\http://{foreman-example-com}/pub/installation_media/sles/15sp3/`.
 . Set the *Operating System Family* to `SUSE` for all SLES systems.
 . Set a location and organization context for the installation media.
+. On the *Parameters* tab, add the `sle-module-basesystem-url` parameter as string with the value `\http://{foreman-example-com}/pub/installation_media/sles/15sp3/`.
```
Comment (melcorr) [inline guides/common/modules/proc_creating-a-suse-operating-system.adoc:18]:
```suggestion
. On the *Parameters* tab, add the `sle-module-basesystem-url` parameter, select the **string** type, and enter the value `\http://{foreman-example-com}/pub/installation_media/sles/15sp3/`.
```

**Review by melcorr — APPROVED** [review]
Hey @maximiliankolb 
Happy to merge as is. IIRC, there are a few selections to be made. 
I'm just hairsplitting but are my suggestions perhaps closer to the user actions? 
Dismiss if necessary. I will not block.

**Comment by maximiliankolb** [comment]
Rebased to master and applied all three suggestions. Ready for re-review.

**Review by melcorr — APPROVED** [review]
Thank you!

**Comment by maximiliankolb** [comment]
Cherry-picked to 3.1: https://github.com/theforeman/foreman-documentation/commit/5cbe1d1ba8703fc2ab5738edce9f5ac9c86bb86b ✅

**Pull request merged**